### PR TITLE
[DOCS] Document encryption key setting needed by Fleet

### PR DESCRIPTION
**Pull request by dedemorton**
Fixes doc changes required by https://github.com/elastic/kibana/issues/69677.

### CODE REVIEW

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 1]
```diff
@@ -148,6 +149,22 @@ that you used to enroll {agent} on your host.
 .. If the secret doesn't match, create a new enrollment token and use the new
 token when you run the `elastic-agent enroll` command.
 
+//TODO: Modify this note when backporting to 7.8 to describe the problem
```
Comment (nchaulet) [inline docs/en/ingest-management/troubleshooting.asciidoc:152]: Do you want to merge the TODO?

Comment (dedemorton) [reply docs/en/ingest-management/troubleshooting.asciidoc:152]: No, I'll remove it before merging. :-)

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 2]
```diff
@@ -148,6 +149,22 @@ that you used to enroll {agent} on your host.
...
+//correctly for that release.
+
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:154]:
```suggestion
```

**Comment by EricDavisX** [comment]
Thanks so much for getting this out @dedemorton - you rock!

**Pull request merged**